src/main/python/ttconv/scc/scc_elements.py

```diff
@@ -194,16 +227,24 @@ def get_origin(self) -> PositionType:
 
   def get_extent(self) -> ExtentType:
     """Computes and returns the current paragraph extent, based on its content"""
-    if len(self.caption_contents):
-      nb_lines = sum(map(lambda br: isinstance(br, SccCaptionLineBreak), self.caption_contents)) + 1
-      text_lengths = [0] + [len(caption_text.text)
-                            for caption_text in self.caption_contents
-                            if isinstance(caption_text, SccCaptionText)]
-      max_text_lengths = max(text_lengths)
+    lines: List[str] = []
```

Comment:
Why is the extent of the paragraph important? In the model, paragraphs do not have extent. Instead, paragraphs flow into a region, which has an extent. I am not convinced that the size of the region should change dynamically. Instead perhaps only the origin of the region could change over time.